source/catalog.html

```diff
@@ -87,7 +70,17 @@ <h2 class="visually-hidden">Корма для кошек</h2>
         <ul class="products__list">
 
           <li class="products__item product">
-            <img class="product__photo" src="img/catalog/product1.png" width="68" height="86" alt="Корм для кошек">
+            <div class="product__photo-wrapper">
+              <picture>
+                <source media="(min-width: 1440px)" width="161" height="172"
+                  srcset="img/catalog/products/desktop/product-1-desktop.png 1x, img/catalog/products/desktop/product-1-desktop@2x.png 2x">
+                <source media="(min-width: 768px)" width="161" height="172"
+                  srcset="img/catalog/products/tablet/product-1-tablet.png 1x, img/catalog/products/tablet/product-1-tablet@2x.png 2x">
+                <img class="product__photo" src="img/catalog/products/mobile/product-1.jpg"
+                  srcset="img/catalog/products/mobile/product-1.jpg 1x, img/catalog/products/mobile/product-1@2x.jpg 2x"
+                  width="68" height="86" alt="Корм для кошек">
+              </picture>
+            </div>
```

Comment:
Можно упростить разметку убрав обёртку, её роль выполнит `picture`
```suggestion
              <picture class="product__photo">
                <source media="(min-width: 1440px)"
                  srcset="img/catalog/products/desktop/product-1-desktop.png 1x, img/catalog/products/desktop/product-1-desktop@2x.png 2x">
                <source media="(min-width: 768px)"
                  srcset="img/catalog/products/tablet/product-1-tablet.png 1x, img/catalog/products/tablet/product-1-tablet@2x.png 2x">
                <img class="product__photo" src="img/catalog/products/mobile/product-1.jpg" 
                  srcset="img/catalog/products/mobile/product-1.jpg 1x, img/catalog/products/mobile/product-1@2x.jpg 2x"
                  width="68" height="86" alt="Корм для кошек">
              </picture>
```

Reply:
👍